### DISCUSSION TITLE
Ability to disable automatically switching workspace when focusing new application

### DISCUSSION THREAD

**Discussion by pmooser** [nikitabobko/AeroSpace#898] [feature-ideas]
I'm a relatively new user of aerospace, and I'm consistently surprised when I go to create a new terminal window, and as soon as I focus my terminal application (typically using spotlight), it switches to a workspace which has an open terminal window. Due to the fact that I use terminal windows in many different workspaces, this is purely an inconvenience, similar to the Spaces feature (which I disable) of switching to a space with open windows when focusing an application. 

I would propose: 

- being able to disable this switching entirely
- being able to disable this switching for specific applications

Right now, it makes it feel like a lot of extra key presses are required for doing something very simple, and it feels rather inefficient to have to manually go back to the workspace I was using when this automatic switching occurs. It typically unfolds like this:

- focus terminal application
- workspace gets automatically switched
- create new terminal window
- move new terminal window to the workspace I was in
- navigate back to that workspace 

It feels really bad to use the application this way. 

If there is already some kind of workaround for this behavior, please let me know.

**Comment by jasoncorso** [comment]
The recommended solution is available in this docs link https://nikitabobko.github.io/AeroSpace/goodies#open-a-new-window-with-applescript

**Reply by pmooser** [reply]
Yes, thank you - I ran across this solution after posting my message, but it is still unfortunate to have to have an applescript snippet plus key binding for each application for which you'd like to be able to do this. I still think there is potential utility in specifically being able to control this behavior.

**Comment by nikitabobko** [comment]
It was asked several times, I don't think it's possible. Keywords to search for related discussions: "focus owner"
